lib/server/capabilities.js

```diff
@@ -93,6 +93,7 @@ var iosCaps = [
 , 'sendKeyStrategy'
 , 'waitForAppScript'
 , 'webviewConnectRetries'
+, 'webkitDebugProxyPort'
```

Comment:
I'm confused as to why this is in with capabilities. Is this not a server argument?